modular_skyrat/modules/pod_locking/pod_locking.dm

```diff
@@ -0,0 +1,27 @@
+/obj/docking_port/mobile/pod/register()
+	. = ..()
+	if(CONFIG_GET(number/minimum_alert_for_pods) != 0)
+		launch_status = NOLAUNCH
+		RegisterSignal(SSsecurity_level, COMSIG_SECURITY_LEVEL_CHANGED, PROC_REF(check_for_evac))
+
+/**
+  * Signal handler for [COMSIG_SECURITY_LEVEL_CHANGED].
+  * Pulls minimum required alert level from modular config and sets pod's [var/launch_status] as [NOLAUNCH] or [UNLAUNCHED] accordingly.
+  * Will not do anything if pod already launched.
+  */
+/obj/docking_port/mobile/pod/proc/check_for_evac(datum/source, new_level)
+	SIGNAL_HANDLER
+
+	var/min_level = CONFIG_GET(number/minimum_alert_for_pods)
+	if(launch_status > UNLAUNCHED)
+		return
+	launch_status = (new_level >= min_level) ? UNLAUNCHED : NOLAUNCH
+
+/obj/machinery/computer/shuttle/pod/emag_act(mob/user)
```

Comment:
Rather than locking this behind an emag, could it simply send a message about the pod launching manually or require some extra confirmation? It'd let antags or opfors bail without having to also get le hacking card

Reply:
I think i could do something like shuttle hijacking so any antag would be able to hack it. Also im not sure if it should even announce such act or not. Or well, send any message at all. Didn't quite got what you mean by that tbh.